source/_includes/integrations/config_flow.md

```diff
@@ -24,17 +23,17 @@ manually:
 {% endif %}
 
 - Browse to your Home Assistant instance.
-- In the sidebar click on _**{% my config icon %}**_.
-- From the configuration menu select: _**{% my integrations %}**_.
+- In the sidebar, select **{% my config icon %}**.
```

Comment:
TIL! Nice, Microsoft describes the reasons for using "selecting" over "clicking" pretty well. Makes a lot of sense.